pkg/ccl/backupccl/backup_processor.go

```diff
@@ -514,34 +506,34 @@ func runBackupProcessor(
 						FileSummaries: make([]roachpb.RowCount, 0),
 					}
 
-					if len(res.Files) > 1 {
+					if len(KVResponse.Files) > 1 {
 						log.Warning(ctx, "unexpected multi-file response using header.TargetBytes = 1")
 					}
 
-					for i, file := range res.Files {
-						f := BackupManifest_File{
+					for i, file := range KVResponse.Files {
+						partialFile := BackupManifest_File{
```

Comment:
calling this a "file" at all is a tad weird; it is just a some data and metadata that identified the span of keys and time that data came from and how much is in it, and isn't a "file" at all just yet, partial or otherwise (it could be complete when we go to write it, for all we know). it could be less confusing to avoid the extra local var completely here: make the `ret` returnedSpan here, _embed_ Manifest_File in it with a comment saying "BackupManifest_File conveniently already has all the metadata fields we need to identify a key/time span" and avoid the word file in this block entirely?

Reply:
Yeah I like that! How's this look?